commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/models/TypeUtils.java

```diff
@@ -34,8 +34,8 @@ public TypeUtils(final Elements elements, final Types types) {
     }
 
 
-    public String getPackageName(final TypeElement typeElement) {
-        return elements.getPackageOf(typeElement).getQualifiedName().toString();
+    public String getPackageName(final Element element) {
```

Comment:
👍 since now where working also with annotations at method level, we can't be restrictive.